proxy/src/neon_api.rs

```diff
@@ -203,9 +218,34 @@ impl NeonApi {
         Ok(U256::from(resp?.used_gas))
     }
 
-    async fn execute(cmd: TaskCommand, ctx: Context) {
-        match cmd {
-            TaskCommand::EstimateGas { tx, response } => {
+    pub async fn emulate(&self, params: TxParams) -> Result<EmulateResponse, NeonError> {
+        let (tx, rx) = oneshot::channel();
+        self.channel
+            .send(Task::new(
+                TaskCommand::Emulate {
+                    tx: params,
+                    response: tx,
+                },
+                None,
+                None,
+            ))
+            .await
+            .unwrap();
+        rx.await.unwrap()
+    }
+
+    pub async fn get_config(&self) -> Result<GetConfigResponse, NeonError> {
+        let (tx, rx) = oneshot::channel();
+        self.channel
+            .send(Task::new(TaskCommand::GetConfig(tx), None, None))
+            .await
+            .unwrap();
+        rx.await.unwrap()
+    }
+
+    async fn execute(task: TaskCommand, ctx: Context) {
+        match task {
+            TaskCommand::EstimateGas { tx, response } | TaskCommand::Emulate { tx, response } => {
```

Comment:
I went with a separate variant just in case there is a need for a different interface/request parameters configuration for `Emulate` in the future. I think we're better off leaving it as it is for now, and revise it later. 

> Additionally, to properly estimate gas, we need both the emulation response and the EVM config used for the evaluation. Therefore, I suggest TaskCommand::Emulate should return a pair: the config and the response

I believe this is out of scope of this PR. And that some config caching should be implemented first.